pandas/io/common.py

```diff
@@ -345,6 +346,10 @@ def get_handle(
         Mode to open path_or_buf with.
     encoding : str or None
         Encoding to use.
+    errors : str, default 'strict'
+        Specifies how encoding and decoding errors are to be handled.
+        See the errors argument for :func:`open` for a full list
```

Comment:
what does this refer to?

Reply:
This is the downstream function used by `to_csv()` to open the file. Collateral damage.